scripts/find.rb

```diff
@@ -0,0 +1 @@
+People.select_all(age: 'age', age: >18)
```

Comment:
You have the right idea, but `select_all` is not a ruby class method. Try using `.where` instead